std/uni.d

```diff
@@ -6421,15 +6456,16 @@ private static struct InputRangeString
 {
     private string s;
 
-    bool empty() @property { return s.empty; }
-    dchar front() @property { return s.front; }
-    void popFront() { s.popFront(); }
+    bool empty() @property { import std.range.primitives : empty; return s.empty; }
```

Comment:
I tried sticking the import at the top of the struct but started getting template instantiation errors.  Would you like me to reindent these?